src/mocking/server/resolvers.ts

```diff
@@ -51,16 +56,57 @@ export const videoResolver: QueryResolver<UniqueArgs, VideoFields> = (obj, args,
   return mirageGraphQLFieldResolver(obj, resolverArgs, context, info)
 }
 
-export const videosResolver: QueryResolver<VideoQueryArgs, GetNewestVideos_videosConnection> = (
-  obj,
-  args,
-  context,
-  info
-) => {
+type VideosResolverConnection =
+  | GetNewestVideos_videosConnection
+  | GetFollowedChannelsRecentVideos_videosConnection
+  | GetFollowedChannelsRecentVideosIds_videosConnection
+
+export const videosResolver: QueryResolver<VideoQueryArgs, VideosResolverConnection> = (obj, args, context, info) => {
+  const { mirageSchema: schema } = context
+  const { where, after, first } = args
+
   const baseResolverArgs = {
-    first: args.first,
-    after: args.after,
+    first,
+    after,
   }
+
+  if (where?.channelId_in?.length && where?.createdAt_gte) {
+    const videos = schema.videos.all().models as VideoModel[]
+    const videosResult = videos
+      .map((v) => ({
+        ...v.attrs,
+      }))
+      .filter((v) => {
+        if (where?.channelId_in?.length) {
+          return where?.channelId_in.includes(v.channelId)
+        }
+        return v
+      })
+      .filter((v) => {
+        if (where?.createdAt_gte) {
+          return new Date(where.createdAt_gte) <= new Date(v.createdAt)
+        }
+        return v
+      })
+      .map((v) => ({
+        ...v,
+        channel: schema.channels.find(v.channelId),
```

Comment:
Are those necessary?

Reply:
Unfortunately, they are. If I delete these lines I'll get no results when `channelIdIn` and `createdAtGte` are passed as an argument.